cg/services/orders/store_order_services/store_microbial_fastq_order_service.py

```diff
@@ -113,6 +113,7 @@ def _create_sample(
             customer=customer,
             sex=SexOptions.UNKNOWN,
             comment=sample_dict["comment"],
+            internal_id=sample_dict["internal_id"],
```

Comment:
To make the tests pass, but this may not solve the problem as it will return None
```suggestion
            internal_id=sample.get("internal_id"),
```

Reply:
Solved using the default None